src/app/public/modules/phone-field/phone-field.component.ts

```diff
@@ -116,24 +116,52 @@ export class SkyPhoneFieldComponent implements OnDestroy, OnInit {
   @Input()
   public set defaultCountry(value: string) {
     if (value && value !== this._defaultCountry) {
+      if (this.supportedCountryISOs && this._userSupportedCountryISOs.indexOf(this.defaultCountry) < 0) {
```

Comment:
Looking at the additional logic makes me wonder if this should be added to the country field component, instead of the phone field, especially since the `supportedCountryISO` is only used by the country field. This way, you could keep the phone field fairly clean with:
```
@Input()
public supportedCountryISOs: string[];
```
And the country field would determine its own requirements.

Another way to think about it: If we put the logic in phone field, wouldn't the country field need the same logic if used in isolation?

Reply:
That’s a good thought. Not sure why I missed that. I’ll look into that.

Reply:
@Blackbaud-SteveBrush I've removed the default country logic here. Lets revisit the country field thoughts in flex to figure out if we need to change something there. This should be ready for review again.